packages/strapi-design-system/src/TimePicker/TimePicker.tsx

```diff
@@ -44,16 +62,41 @@ export const TimePickerInput = ({
    * @deprecated This is no longer used.
    */
   selectButtonTitle: _selectButtonTitle,
+  textValue: textValueProp,
+  defaultTextValue = '',
+  onTextValueChange,
+  value: valueProp,
+  defaultValue,
```

Comment:
Just out of curiosity: why is that needed if you have `useControllableState`? I thought for the other inputs we only needed `value`.

Reply:
If you're controlling the value you might need to control the text value e.g. with relations we clear the field. These were passed straight to the combobox but i've had to expose them because i'm controlling the input internally to format it correctly :)